### PR TITLE
initial commit data-PhysicsTools-NanoAOD.spec

### PR DESCRIPTION
**Pull request by mrodozov**
initial commit for adding data-PhysicsTools-NanoAOD.spec file and a reference for it in cmsswdata.spec

### CODE REVIEW

File: cmsswdata.spec [hunk 1]
```diff
@@ -46,6 +46,7 @@ Requires: data-SimTracker-SiStripDigitizer
 Requires: data-CalibCalorimetry-EcalTrivialCondModules
 Requires: data-DataFormats-PatCandidates
 Requires: data-SimTransport-HectorProducer
+Requires: data-PhysicsTools-NanoAOD.spec
```
Comment (smuzaffar) [inline cmsswdata.spec]: @mrodozov , please remove the .spec extension  here

Comment (mrodozov) [reply cmsswdata.spec]: @smuzaffar updated

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by davidlange6** [comment]
merge

**Pull request merged**